### DISCUSSION TITLE
How To Fix Failed To Load Module Script: Expected A Javascript Module Script But The Server Responded With A Mime Type Of “text/plain”. Strict Mime Type Checking Is Enforced For Module Scripts Per Html Spec

### DISCUSSION THREAD

**Discussion by amirtbi** [vercel/vercel#7726] [Help]
Hi,
I am deploying a Vuejs App with Vite  and a node js server in Vercel. After the production is done, when the page is loaded, the fail to load module script is appeared in console!. how can I solve this issue?

**Comment by pablodz** [comment]
x2

![image](https://github.com/vercel/vercel/assets/28235457/650f8ae9-fb93-46b6-8b8f-c060428d95d1)

**Reply by Vietanh30** [reply]
Have you fixed it yet? Please let me know.

**Reply by pablodz** [reply]
create a file vercel.json with config @Vietanh30 

```
{
    "rewrites": [
        {
            "source": "/(.*)",
            "destination": "/"
        }
    ]
}
```

and if you are using vite, add the base option when build ` vite build --base=/  `
